app/models/katello/content_view_version.rb

```diff
@@ -46,7 +46,7 @@ class ContentViewVersion < Katello::Model
 
     validates_lengths_from_database
 
-    validates :minor, :uniqueness => {:scope => [:content_view_id, :major], :message => N_(", must be unique to major and version id version.")}
+    validates :minor, :uniqueness => {:scope => [:content_view_id, :major], :message => "CV Version with same major and minor version already exists."}
```

Comment:
I think we still want the internalization here (`N_(`)

Reply:
I haven't been able to look at this yet, but is there a way to keep rails from prepending 'Minor' to the message?

Reply:
@beav I tried a few things with rails 5 syntax and still no good, from things I tried manually or trying things from different articles. It shows the correct error message for the 2nd message but the first one is still shows `minor`